lib/command/utils/error.js

```diff
@@ -1,24 +1,62 @@
 'use strict';
 
+const _ = require('lodash');
+
+const DOCS_LINE =
+  'See https://www.contentful.com/developers/docs/references/errors for more information.';
+
 exports.format = function (command) {
-  return function (msg) {
-    if (typeof msg === 'object' && msg.request) {
-      return serverError(msg);
+  return function (res) {
+    if (typeof res === 'object' && res.request) {
+      return serverError(res);
     } else {
-      return textError(msg);
+      return textError(res);
     }
   };
 
-  function textError (msg) {
-    return `Failed to ${command} the widget: ${msg.toString()}`;
+  function textError (err) {
+    return `Failed to ${command} the widget: ${err.toString()}`;
   }
 
-  function serverError (msg) {
-    let seeDocs =
-      'See https://www.contentful.com/developers/docs/references/errors for more information';
-    let errorMessage = msg.error.message || msg.error.id;
+  function serverError (res) {
+    const err = processServerError(res);
 
-    return (`${msg.request.method.toUpperCase()} request failed ` +
-      `(${msg.error.sys.id} Error)\n${errorMessage}\n${seeDocs}`);
+    return `${err.method} (${command}) ` +
+      `request failed because of ${err.id} error.` +
+      `\n${err.reasons.join('\n')}\n${DOCS_LINE}`;
```

Comment:
It doesn't look that nice with `return` and function around (you need to keep it unindented) and it adds extra new lines before and after the message.